docs/remote/containers-tutorial.md

```diff
@@ -136,17 +128,25 @@ The `devcontainer.json` is basically a config file that determines how your dev
 {
     "name": "Node.js Sample",
     "dockerFile": "Dockerfile",
-    "appPort": 3000,
-    "extensions": [
-        "dbaeumer.vscode-eslint"
-    ],
+
+    // Use 'settings' to set *default* container specific settings.json values on container create. 
     "settings": {
         "terminal.integrated.shell.linux": "/bin/bash"
     },
+
+    // Add the IDs of extensions you want installed when the container is created in the array below.
+    "extensions": [
+        "dbaeumer.vscode-eslint"
+    ],
+    
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
```

Comment:
Looks like an extra space or two got here.

Reply:
You were too quick for me! I ended up removing the comments as you suggested originally since we have the table below the devcontainer.json definition explaining the various properties. I also updated that table to match the properties in the newer version of devcontainer.json we're now including. 

Let me know what you think of that update, or if you think the original version with comments and the other table was better.

Reply:
That works too!